modules/oe_theme_content_publication/templates/oe-theme-content-publication-description.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * Theme template for Publication description extra field.
+ *
+ * @see \Drupal\oe_theme_content_publication\Plugin\ExtraField\Display\PublicationDescription::viewElements()
+ */
+#}
+<div class="ecl-row">
```

Comment:
We can hide the whole row if both are empty.

Reply:
PublicationDescription::viewElements returns empty array if there aren't values so this template isn't used.

Reply:
right1